exercises/es/exc_02_13.py

```diff
@@ -1,21 +1,21 @@
 import spacy
 from spacy.matcher import Matcher
 
-nlp = spacy.load("en_core_web_sm")
+nlp = spacy.load("es_core_news_sm")
 doc = nlp(
-    "Twitch Prime, the perks program for Amazon Prime members offering free "
-    "loot, games and other benefits, is ditching one of its best features: "
-    "ad-free viewing. According to an email sent out to Amazon Prime members "
-    "today, ad-free viewing will no longer be included as a part of Twitch "
-    "Prime for new members, beginning on September 14. However, members with "
-    "existing annual subscriptions will be able to continue to enjoy ad-free "
-    "viewing until their subscription comes up for renewal. Those with "
-    "monthly subscriptions will have access to ad-free viewing until October 15."
+    "Cuando Pac-Man debutó en Tokio, en 1980, nadie podría haber predicho "
+    "que se convertiría en el videojuego más exitoso de todos los tiempos. Hoy, "
+    "40 años después, aun sigue sorprendiendo. Su desarrolladora, Bandai Namco, "
+    "ha anunciado novedades en el marco del aniversario del juego. La celebración "
+    "del 40 aniversario de Pac-man en 2020 incluirá el début de una nueva canción "
+    "temática, compuesta por el famoso artista japonés de Techno Ken Ishii. "
+    "Además de estas novedades, Bandai Namco publicará nuevas versiones del videojuego."
+    "La primera será Pac-Man Live Studio, en Twitch, en colaboración con Amazon Games."
```

Comment:
This is very cool! 💯 🤩 Glad to see that this works so well in Spanish. For German, I had to leave this in English.

Reply:
Aaahh...this one was really hard to get right! 😅 But thankfully it was Pac-man's anniversary around the time I was doing it! 😂